python/GafferUI/NameValuePlugValueWidget.py

```diff
@@ -53,14 +53,11 @@ class NameValuePlugValueWidget( GafferUI.PlugValueWidget ) :
 
 	def __init__( self, childPlug ) :
 
-		if isinstance( childPlug, Gaffer.Plug ) :
-			childPlugs = { childPlug }
-		else :
-			childPlugs = childPlug or set()
-
 		self.__row = GafferUI.ListContainer( GafferUI.ListContainer.Orientation.Horizontal, spacing = 4 )
 
-		GafferUI.PlugValueWidget.__init__( self, self.__row, childPlugs )
+		GafferUI.PlugValueWidget.__init__( self, self.__row, childPlug )
+
+		childPlugs = self.getPlugs()
```

Comment:
`childPlugs` is extremely misleading, as is the pre-existing constructor argument called `childPlug`. They both refer to the plugs being shown by this widget, and not the children at all, don't they? My personal taste would probably be to repeat the very cheap `getPlugs()` call below so I don't have to track another variable, but let's at least call the variable `plugs`. Could you also open a PR on master once this is merged, to rename the `childPlug` constructor argument to just `plug`?

Reply:
Updated to `getPlugs()` [here](https://github.com/GafferHQ/gaffer/pull/3980/commits/595c1d38ad6361f871bd4d46c5c87ce8294b4224).